src/components/Git/GitAccountsSlice/gitThunk.ts

```diff
@@ -0,0 +1,12 @@
+import { createAsyncThunk } from "@reduxjs/toolkit"
+import { gitApi } from "../../../API/gitApi"
+import { AccountType } from "./gitAccountSlice"
+
+export const gitThunk = createAsyncThunk(
+    "account/gitAccount",
+    // Declare the type your function argument here:
+    async (data: AccountType) => {
+        let res = gitApi.getAccount(data.nik_name)
```

Comment:
there is no async handler, please handle the request first, get some data, build a payload and return relevant object